websites/P/PayPal/presence.ts

```diff
@@ -0,0 +1,126 @@
+const presence = new Presence({
+    clientId: "924713441464295424"
+  }),
+  browsingTimestamp = Math.floor(Date.now() / 1000);
+
+let title: HTMLElement;
+presence.on("UpdateData", async () => {
+  const presenceData: PresenceData = {
+    largeImageKey: "logo",
+    startTimestamp: browsingTimestamp
+  };
+
+  if (document.location.pathname === "/myaccount/summary") {
+    presenceData.details = "Viewing";
+    presenceData.state = "Dashboard";
```

Comment:
For all your `details` and `state`s, only use two lines where the text will be truncated. Otherwise, just use `details